layouts/partials/head.html

```diff
@@ -0,0 +1,13 @@
+{{ if  or (eq .Section "posts") (eq .Section "tags") }}
```

Comment:
I'm not sure whether this is sufficient. Let's discuss.

Reply:
Yes, we will need to update it. I was planning to bring everything over so the site would work and then clean up in a future PR.

Reply:
Yes, we will need to update it. I was planning to bring everything over so the site would work and then clean up in a future PR.